### ISSUE TITLE
How about when you create or delete a table in voyager admin it will create migration file too ?

### ISSUE THREAD

**Issue by rizkiheryandi** [thedevdojo/voyager#3668]
## Version information

 - Laravel: v5.7.0
 - Voyager: v1.1
 - PHP: 7.1
 - Database: 5.7

## Description of problem

Hi all, have you ever thought  about making or delete the table in voyager admin GUI it simultaneously make the migration file too ? it's cool right ? 

It's just for keep the database development history under our control :D

## Proposed solution
If the voyager community has the same mind with me, maybe i will help and create a pull request  

## Additional context
It's my oppinion, if something about my "feature request" goes worng, forgive me :) THANKS !!

**Comment by rizkiheryandi** [comment]
Wow, im new to using voyager, but i got this picture when access other issue in github

![29519448-ce9ea3d4-867d-11e7-98b7-5ef39b3d6493](https://user-images.githubusercontent.com/25028868/47268441-3dd71680-d57b-11e8-996c-8aacc9ca0736.png)

It's look like in previous version, voyager has feature for "make migration file" too when create a table

But, why there's no this feature in 1.1 ?

Whats wrong ? 

:thinking:

**Comment by fletch3555** [comment]
We removed that functionality a long time ago, since all it did was basically call `artisan make:migration`, which only creates a blank migration file which caused more confusion than it's benefit.  We've determined it's too complex a task for us to undertake given our existing workload, so we've left it up to the community to provide a means of generating and populating migration files.  This already has a number of issues here.
